### ISSUE TITLE
mythbox.log not created on windows - KeyError

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#16]
```
mythbox.log is never created on windows because of this key error.

23:22:24 T:1600 M:2378854400  NOTICE: MythBox: loggerIniFile = C:\Program
Files\XBMC-SVN17762-jester\scripts\trunk\mythbox_log.ini
23:22:24 T:1600 M:2378838016  NOTICE: Traceback (most recent call last):
23:22:24 T:1600 M:2378838016  NOTICE:   File
"special:\\xbmc\system\python\python24.zlib\logging\config.py", line 151,
in fileConfig
23:22:24 T:1600 M:2378838016  NOTICE: KeyError: 'logfile'
23:22:24 T:1600 M:2378833920  NOTICE: DEBUG | default.py | Line 94 |
Mythbox Logger Initialized


```

Original issue reported on code.google.com by `semir.pa...@gmail.com` on 17 Feb 2009 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
Also appears on my Apple TV so not limited to Windows. Though I am getting 
messages through the main xbmc.log instead so not a big issue.
```

Original comment by `joshua.k...@gmail.com` on 31 Oct 2010 at 3:33

**Comment by GoogleCodeExporter** [comment]
```
I think this will fix itself when xbmc upgrades to python 2.6
```

Original comment by `semir.pa...@gmail.com` on 26 Nov 2010 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
All logging output now goes to xbmc.log.  

mythbox.log is no more.
```

Original comment by `semir.pa...@gmail.com` on 30 Dec 2011 at 8:28
- Changed state: **WontFix**
